diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..b83d222
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+/target/
diff --git a/src/main/java/Buisness/Main.java b/src/main/java/Buisness/Main.java
index 0f0057f..92a30c2 100644
--- a/src/main/java/Buisness/Main.java
+++ b/src/main/java/Buisness/Main.java
@@ -4,7 +4,7 @@
  */
 package Buisness;
 
-import access.xmlread;
+import controller.xmlread;
 import com.mycompany.beadando.resources.Book;
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.Path;
diff --git a/src/main/java/access/books.xml b/src/main/java/controller/books.xml
similarity index 100%
rename from src/main/java/access/books.xml
rename to src/main/java/controller/books.xml
diff --git a/src/main/java/access/xmlread.java b/src/main/java/controller/xmlread.java
similarity index 99%
rename from src/main/java/access/xmlread.java
rename to src/main/java/controller/xmlread.java
index 14a69c6..a5fe37d 100644
--- a/src/main/java/access/xmlread.java
+++ b/src/main/java/controller/xmlread.java
@@ -2,7 +2,7 @@
  * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
  * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
  */
-package access;
+package controller;
 
 import com.mycompany.beadando.resources.Book;
 import com.mycompany.beadando.resources.Borito;
diff --git a/src/main/java/access/xmlwrite.java b/src/main/java/controller/xmlwrite.java
similarity index 98%
rename from src/main/java/access/xmlwrite.java
rename to src/main/java/controller/xmlwrite.java
index 25b52db..ac13cc4 100644
--- a/src/main/java/access/xmlwrite.java
+++ b/src/main/java/controller/xmlwrite.java
@@ -2,7 +2,7 @@
  * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
  * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
  */
-package access;
+package controller;
 import com.mycompany.beadando.resources.Book;
 import org.w3c.dom.Document;
 import org.w3c.dom.Element;
diff --git a/src/main/java/access/books.xml b/src/main/resources/books.xml
similarity index 100%
copy from src/main/java/access/books.xml
copy to src/main/resources/books.xml
